ui/job-view/pushes/JobButton.jsx

```diff
@@ -134,6 +137,13 @@ export default class JobButtonComponent extends React.Component {
             title="classified"
           />
         )}
+        {possiblyIntermittent && (
+          <FontAwesomeIcon
+            icon={faMitten}
+            className="possibly-intermittent-icon"
+            title="possibly intermittent"
```

Comment:
Please change to `Intermittent failure - There is a successful run of this task for the same push.`